animations/staggered-animations/index.md

```diff
@@ -0,0 +1,383 @@
+
+This guide provides two staggered animation examples,
+though only one is described here. The second, more complex example,
+is available for your reference.
+
+[basic_staggered_animation](https://github.com/flutter/website/tree/master/_includes/code/animation/basic_staggered_animation)
+: Shows a series of sequential and overlapping animations on a single widget.
+  Clicking in the screen begins an animation that changes opacity, size,
+  shape, color and padding.
+
+[staggered_pic_selection](https://github.com/flutter/website/tree/master/_includes/code/animation/staggered_pic_selection)
+: Shows deleting an image from a list of images displayed in one of three sizes.
+  This example uses two animation controllers: one for image selection/de-selection,
+  and one for image deletion. The selection/de-selection animation is staggered.
+  (To see effect this yourself, you may need to increase the ```timeDilation``` value.)
+  Select one of the largest images which shrinks while displaying a checkmark
+  inside a blue circle. Then select one of the smallest images. The large image
+  expands before the small image shrinks. This behavior is similar to that used
+  by Google Photos.
+
+</aside>
+
+The following video demonstrates a simple staggered animation
+described in this guide:
+
+<!--
+  Use this instead of the default YouTube embed code so that the embed
+  is responsive.
+-->
+<div class="embed-container"><iframe src="https://www.youtube.com/embed/0fFvnZemmh8?rel=0" frameborder="0" allowfullscreen></iframe></div>
+
+In the video, you see the following animation on a single widget,
+which begins as a bordered blue square with slightly rounded corners.
+The square:
+
+* Fades in
+* Widens
+* Becomes taller while moving upwards
+* Transforms into a bordered circle
+* Changes color to orange
+
+After running forwards, the animation runs in reverse.
+
+* TOC Placeholder
+{:toc}
+
+<aside class="alert alert-info" markdown="1">
+**New to Flutter?**
+You should first understand how to create a layout using Flutter’s
+widgets.  For more information, see [Building Layouts in
+Flutter](/tutorials/layout/).
+</aside>
+
+**Basic structure of a staggered animation**<br>
+
+<div class="whats-the-point" markdown="1">
+
+<b> <a id="whats-the-point" class="anchor" href="#whats-the-point" aria-hidden="true"><span class="octicon octicon-link"></span></a>What's the point?</b>
+
+* Break the animation down into Intervals.
+* Regardless of how long the animation lasts in real time,
+  Intervals are specified within the range of 0.0 and 1.0.
+  For example, an Interval might run from 0.5 to 0.55.
+* Create an Animation object for each Interval.
+* Create a Tween for each property that animates in an interval.
+  The Tween object specifies the start and end values for that property.
+  For example, a width tween might specify a starting width of 50 pixels
+  and an ending width of 150.
+</div>
+
+The following diagram shows the Intervals used in the
```

Comment:
This chart is great (although there's some extra white space before the last two headings).

I think it would help to explain how the chart relates to the code a bit. The app is essentially just animating a Container whose decoration and size are animated. It's within another Container whose padding moves the inner container around and an Opacity widget that's used to fade everything in and out.